e2e/plugin/trust_test.go

```diff
@@ -44,14 +44,17 @@ func TestInstallWithContentTrust(t *testing.T) {
 
 	icmd.RunCommand("docker", "plugin", "rm", "-f", pluginName).Assert(t, icmd.Success)
 
-	result = icmd.RunCmd(icmd.Command("docker", "plugin", "install", "--grant-all-permissions", pluginName),
-		fixtures.WithConfig(dir.Path()),
-		fixtures.WithTrust,
-		fixtures.WithNotary,
-	)
-	result.Assert(t, icmd.Expected{
-		Out: fmt.Sprintf("Status: Downloaded newer image for %s@sha", pluginName),
-	})
+	// TODO(krissetto): this command seems to panic the engine, needs to be checked
+	// `panic: interface conversion: reference.canonicalReference is not reference.Tagged: missing method Tag`
```

Comment:
this panic occurs [here](https://github.com/moby/moby/blob/master/plugin/fetch_linux.go#L203)

Reply:
[moby PR](https://github.com/moby/moby/pull/47299) related to this `docker plugin install` test that fails and is currently skipped